src/content/app/tools/blast/components/blast-settings/BlastSettings.tsx

```diff
@@ -149,25 +151,32 @@ const BlastSettings = ({ config }: Props) => {
           <h1 className={styles.title}>Blast</h1>
           <div className={styles.runJob}>Run a job</div>
           <div>
-            {buildSelect({
-              ...(config.parameters['database'] as BlastSelectSetting),
-              selectedOption: blastParameters.database as string,
-              onChange: onDatabaseChange
-            })}
+            <BlastSelect
+              options={config.parameters.database.options as Option[]}
+              label={config.parameters.database.label}
+              description={config.parameters.database.description as string}
```

Comment:
no need to pass description here (and for other settings) that we are not using the description